pyproject.toml

```diff
@@ -5,10 +5,11 @@ description = ""
 authors = ["Our World in Data <tech@ourworldindata.org>"]
 
 [tool.poetry.scripts]
+etl = 'apps.cli:cli'
 etlcli = 'apps.cli:cli'
 etlwiz = 'apps.wizard.cli:cli'
 etl-wizard = 'apps.wizard.cli:cli'
-etl = 'etl.command:main_cli'
+etlr = 'etl.command:main_cli'
```

Comment:
Is this just a short version of `etl run` or does it have some other use? (e.g. lower overhead than `etl run`)